tests/API/XCCDF/unittests/test_report_anaconda_fixes_ds.sh

```diff
@@ -45,6 +65,14 @@ grep "$line1" $result
 grep "$line2" $result
 grep "$line3" $result
 grep -v "$line1" $result | grep -v "$line2" | grep -v "$line3"
-[ "`grep -v "$line1" $result | grep -v "$line2" | grep -v "$line3" | sed 's/\W//g'`"x == x ]
+
+case $(uname) in
```

Comment:
How about adding `xsed` function to the `test_common.sh` and using `sed` or `gsed` there based on `uname`?